service/resource/cloudformationv2/adapter/launch_configuration.go

```diff
@@ -31,14 +32,14 @@ type BlockDeviceMapping struct {
 	VolumeType          string
 }
 
-func (l *launchConfigAdapter) getLaunchConfiguration(customObject awstpr.CustomObject, clients Clients) error {
+func (l *launchConfigAdapter) getLaunchConfiguration(customObject v1alpha1.AWSConfig, clients Clients) error {
 	if len(customObject.Spec.AWS.Workers) == 0 {
 		return microerror.Mask(invalidConfigError)
 	}
 
-	l.ImageID = customObject.Spec.AWS.Workers[0].ImageID
-	l.InstanceType = customObject.Spec.AWS.Workers[0].InstanceType
-	l.IAMInstanceProfileName = keyv1.InstanceProfileName(customObject, prefixWorker)
+	l.ImageID = keyv2.WorkerImageID(customObject)
+	l.InstanceType = keyv2.WorkerInstanceType(customObject)
```

Comment:
Little bit of boyscouting to use the key helpers. Otherwise this is all renaming.